Library/Formula/mysqltuner.rb

```diff
@@ -0,0 +1,13 @@
+class Mysqltuner < Formula
+  url "https://github.com/major/MySQLTuner-perl/archive/v1.4.0.tar.gz"
+  sha256 "8d5a03b64da2164c6bd93b79700c93db088a14155bcc8cb63c65d049909d793e"
+  homepage "http://mysqltuner.com"
+
+  def install
+    bin.install "mysqltuner.pl" => "mysqltuner"
+  end
+
+  test do
+    system "mysqltuner", "--help"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!